requirements.txt

```diff
@@ -1,11 +1,20 @@
-netaddr==0.7.19
-argparse==1.2.1
+asn1crypto==0.24.0
+bcrypt==3.1.6
+cffi==1.12.3
+configobj==5.0.6
+cryptography==2.6.1
+enum34==1.1.6
 funcsigs==1.0.2
-paramiko==1.17.6
+ipaddress==1.0.22
```

Comment:
`enum34` and `ipaddress` are only needed with python 2.7. Should we make these requirements conditional?

Reply:
Sounds good, will do!

Reply:
done in 37e544521ddd9d821148e53455054bbbd368dcad